src/java/org/apache/cassandra/tools/NodeTool.java

```diff
@@ -144,7 +143,6 @@ public int execute(String... args)
                 GetInterDCStreamThroughput.class,
                 GetLoggingLevels.class,
                 GetMaxHintWindow.class,
-                GetMinimumKeyspaceRF.class,
```

Comment:
Please update `NEWS.txt` removing reference to this since you are removing

Reply:
I think no entry was added to `NEWS.txt`. We should however mention the new guardrail in that file, adding it to [this list](https://github.com/apache/cassandra/blob/0965710345b1e9887d6bbd53b3df45278f8743f4/NEWS.txt#L68-L87).